sycl/doc/extensions/supported/sycl_ext_intel_dataflow_pipes.asciidoc

```diff
@@ -110,9 +110,9 @@ A pipe type is a specialization of the pipe class:
 
 [source,c++,Pipe type def,linenums]
 ----
-template <typename name,
-          typename dataT,
-          size_t min_capacity = 0>
+template <typename _name,
+          typename _dataT,
+          size_t _min_capacity = 0>
```

Comment:
That recent change only made the documentation consistent with [the current released header file](https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/ext/intel/pipes.hpp). And this "template parameters with leading underscores" style is widely used under [sycl/include/sycl/ext/intel](https://github.com/intel/llvm/tree/sycl/sycl/include/sycl/ext/intel).

Do you think this is a blocking issue to this patch? What's your suggestion here?

Reply:
I'd like to keep a consistent style for the extension specification documentation.  There is no strict requirement that the naming of the template parameters in the specification matches the name in the implementation header, so it is OK if they are not the same.  Since this PR is changing the names in the specification, I'd like that to be fixed in this PR.

I understand that you have been fixing some things in the specs that were problems even before you started this PR, and I appreciate that.  However, this seems like a case where the PR is making something worse, so I'd like to address it in this PR.

FWIW, I looked at several of the extension headers and none of the non-FPGA headers I looked at had the initial underscore style.  Perhaps this is a common style in the FPGA team, but it does not match the style I see in the rest of DPC++.

Reply:
I see. It's good to know that it's not necessary to keep the naming consistent in the documentations and in the headers. In that case, I'll only update the documentations in this PR to meet your suggested style.

Reply:
Thanks.